.vulture_whitelist.py

```diff
@@ -11,6 +12,7 @@
 PermissionValue.CR
 
 BOLD_GREEN
+YELLOW
```

Comment:
Instead of keeping the unused YELLOW in the code, it might make more sense to keep the unused DspToolsUserInfo. Of course only if we anticipate it being used in the future. 

If you think we won't use DspToolsUserInfo in the future, I would remove YELLOW too, because there is no reason to keep it.